.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,60 @@
+name: Bug Report
+description: Submit a bug report
+labels: [needs triage]
+body:
+  - type: textarea
+    attributes:
+      label: Description
+      description: A clear and concise description of what the bug is, and why you consider it to be a bug.
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Issue type
+      options:
+        - Bug
+        - Question
+        - Feature request
```

Comment:
Do we really need this dropdown? I guess there is a separate template for feature requests

Reply:
hm, yeah, maybe it is redundant, will remove it then